src/components/AnchorifyText.js

```diff
@@ -37,7 +37,7 @@ class AnchorifyText extends React.Component {
                     if (React.Children.count(this.props.children) === 1) {
                       return React.cloneElement(this.props.children, {url: t, key: key});
                     } else {
-                      if (t.substr(0, 7) !== 'http://' && t.substr(0, 8) !== 'https://') t = 'http://' + t;
+                      if (t.indexOf('://') === -1) t = 'http://' + t;
                       return (<a key={key} href={t} target={this.props.target}>{t}</a>);
```

Comment:
It is better to not modify rendering `{t}`.
We should only fix protocol on `{t}` of `href`.